main/ahb_for_sles.go

```diff
@@ -97,13 +98,49 @@ func parseCfg(filename string) (map[string]map[string]string, error) {
 	return ini, nil
 }
 
-func _isRegistered() (bool, error) {
-	services, err := filepath.Glob("/etc/zypp/services.d/*.service")
-	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		return false, err
+func _getSUSEConnectStatus() (bool, bool, error) {
```

Comment:
@jesusbv, please ensure that all error events are getting logged in the telemetry.
Eg: ext.ExtensionEvents.LogErrorEvent(ENABLE_EVENT,fmt.Sprintf(OPERATION_FAILURE_MSG, ENABLE_EVENT, err.Error()))

Using object ext.ExtensionEvents, you should be able to send important telemetry to Azure Kusto. This is to ensure much ease in debugging any future errors that customers might face and correspondingly witness how the extension is performing.